src/components/Chart/DonutChart.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import ClassNames from 'classnames';
+import { findDOMNode } from 'react-dom';
+
+require('patternfly/dist/js/patternfly-settings');
+require('patternfly/dist/js/patternfly-settings-charts');
+
+const c3ChartDefaults = window.patternfly.c3ChartDefaults();
+const pfSetDonutChartTitle = window.patternfly.pfSetDonutChartTitle;
+
+/**
+ * DonutChart Component for Patternfly React
+ */
+class DonutChart extends React.Component {
+  componentDidMount() {
```

Comment:
note that this will not rerender on props change, maybe have a look at https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/charts/Chart.js

Reply:
added the missing `DidUpdate` method